04.5-rmarkdown.Rmd

```diff
@@ -0,0 +1,415 @@
+# R Markdown Reports
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Learning objectives are to be able to: Explain why R Markdown files are useful for creating data analysis reports. Recognize how to test pieces of code within R Markdown files. Recognize how to create a full R Markdown file report in html and other formats. Explain that R Markdowns generate reports from code in a way that is a bit closer to someone else trying your code."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20eecbcf66d_84_0")
+```
+
+## Reports support reproducibility
+
+Using R Markdown files helps you to create supports that can more transparently show what you did for your analysis and it can help you to test that your code works as expected. Scripts allow you to save code, but they do not allow you to have the following additional benefits. 
+
+The following are reasons why R Markdown files help reproducibility:
+
+- They allow you to show and share your code and the output of your code (this can be done in several ways depending on what you want)
```

Comment:
I think the easiest way to show the benefit is to show a rendered report. It might be good if we have an image and/or link to a mock up rendered markdown to show how cool it is. Maybe side by side with the Rmarkdown writing